autorest/black/__init__.py

```diff
@@ -26,14 +26,27 @@ def __init__(self, autorestapi):
             output_folder_uri = output_folder_uri[3:]
         self.output_folder = Path(output_folder_uri)
 
-    def process(self) -> bool:
-        # apply format_file on every file in the output folder
+    def proc_files(self, folder: Path, pattern: str):
         list(
             map(
                 self.format_file,
-                [f for f in self.output_folder.glob("**/*") if f.is_file()],
+                [f for f in folder.glob(pattern) if f.is_file()],
             )
         )
+
+    def process(self) -> bool:
+        # apply format_file on every file in the output folder
+        self.proc_files(folder=self.output_folder, pattern="**/*")
+
+        # format files that may be outside output folder(setup.py, etc)
```

Comment:
we shouldn't go outside of `output-folder`, same reason for why we can't generate code outside of `output-folder`

Reply:
make sense. In the future we will have `clear-python-folder` and we can avoid similar problem